### PR TITLE
Se dims and hidden method

### PR DESCRIPTION
**Pull request by esheldon**
Add feature where user can force an exact se_dim (single epoch image dim)

Also add hidden method to DM psfs that gets called

### CODE REVIEW

File: descwl_shear_sims/psfs/dmpsfs.py [hunk 1]
```diff
@@ -78,14 +78,33 @@ def computeImage(self, image_pos):  # noqa
...
+        color: afw_image.Color
+            A color, which is ignored
+        """
+
+        return self._doComputeKernelImage(
+            image_pos=image_pos,
+            color=color,
+        )
+
+    def _doComputeKernelImage(self, image_pos, color=None):  # noqa
```
Comment (beckermr) [inline descwl_shear_sims/psfs/dmpsfs.py:98]: Is there DM code that accesses hidden methods?

Comment (esheldon) [reply descwl_shear_sims/psfs/dmpsfs.py:98]: Yes :(

Comment (beckermr) [reply descwl_shear_sims/psfs/dmpsfs.py:98]: Ouch. We should file a ticket if you have the details. This is bad form!

Comment (esheldon) [reply descwl_shear_sims/psfs/dmpsfs.py:98]: agreed

Comment (esheldon) [reply descwl_shear_sims/psfs/dmpsfs.py:98]: Its hard to track down because it happens in C++ and the names are different there

Comment (esheldon) [reply descwl_shear_sims/psfs/dmpsfs.py:98]: Here is an official test where they do it

https://github.com/lsst/afw/blob/master/tests/test_psf_trampoline.py#L66

Comment (esheldon) [reply descwl_shear_sims/psfs/dmpsfs.py:98]: If that is a must override it should be part of the public API

**Comment by esheldon** [comment]
Shall we merge anyway?

**Review by beckermr — APPROVED** [review]
Yes we should merge!

**Pull request merged**